### DISCUSSION TITLE
Modes and documentation

### DISCUSSION THREAD

**Discussion by sjLambda** [qutebrowser/qutebrowser#8254] [Q&A]
Are all the modes documented someplace? I looked in all the usual places, I see them mentioned here and there, but I was wondering if they are described properly any place. I am especially interested in switching modes. Are there any restricitions when switching from one to the other. I got into this because I was trying to escape from youtube page that traps the keyboard in one mode and I can't seem to escape to hints.

**Comment by plum** [comment]
Well, as a long time user, I'd suggets the answer  is not really. 

For example there is mention in the dcocumenttion 'only available in prompt mode' but i see no other mention of such a mode or how to get into it.  I assume it means the command prompt, button , I get the warning 'only available in prompt mode' for ` :prompt-open-download --pdfjs` exploring the meaning of this by attempted efforts, I find tha the 'prompt mode' cannot be entrered into manually by the user'. In other words, its just the terminlogy for any popup prompt, such as a java download prompt.

To be fair, quteborwser is  fairly well documented overall,  and a good deal of hard work has gone into that.  Consider how difficult it is to document this software clearly. I think a good job as been done, but it does take repeated study, nontheless.
But I agree with you, a short page dedicated to overview the 'modes' would be useful.
